File: UI/LuminaUI/src/app/shared/Models/BillPAList.ts
export class BillPAList {
    public _id: string;
    public ProductID: string;
    public Category: string;
    public Color: string;
    public Count: number;
    public Sizes: string;
    public LastPurchasePrice: number;
    public estimatedTotalPrice: number;
    public EstimatedTotalPrice: number;
    public CategoryID: number;
  
    constructor(
      _id: string,
      productID: string,
      category: string,
      color: string,
      count: number,
      sizes: string,
      lastPurchasePrice: number,
      estimatedPrice: number,
      estimatedTotalPrice: number,
      categoryID: number
    ) {
      this._id = _id;
      this.ProductID = productID;
      this.Category = category;
      this.Color = color;
      this.Count = count;
      this.Sizes = sizes;
      this.LastPurchasePrice = lastPurchasePrice;
      this.estimatedTotalPrice = estimatedPrice;
      this.EstimatedTotalPrice = estimatedTotalPrice;
      this.CategoryID = categoryID;
    }
  }
  

File: API/LuminaAPI/Model/PRMS/BillPAList.cs
namespace LuminaAPI.Model.PRMS
{
    public class BillPAList
    {
        public string _id {  get; set; }

        public string ProductID { get; set; }

        public string Category { get; set; }

        public string Color { get; set; }

        public int Count { get; set; }

        public string Sizes { get; set; }

        public int LastPurchasePrice { get; set; }

        public int EstimatedPrice { get; set; }

        public int EstimatedTotalPrice { get; set; }

        public int CategoryID { get; set; }
    }
}
